contributors/design-proposals/pod-security-policy.md

```diff
@@ -0,0 +1,351 @@
+## Abstract
+
+PodSecurityPolicy allows cluster administrators to control the creation and validation of a security
+context for a pod and containers. The intent of PodSecurityPolicy is to protect the cluster from the
+pod and containers, not to protect a pod or containers from a user.
+
+## Motivation
+
+Administration of a multi-tenant cluster requires the ability to provide varying sets of permissions
+among the tenants, the infrastructure components, and end users of the system who may themselves be
+administrators within their own isolated namespace.
+
+Actors in a cluster may include infrastructure that is managed by administrators, infrastructure
+that is exposed to end users (builds, deployments), the isolated end user namespaces in the cluster, and
```

Comment:
Can you define "builds, deployments"? I assume these are components of some kind of CI/CD system, but since they're not native Kubernetes concepts, people might not be familiar with what they do, who they do it for, and what privileges they require. It sounds like they are some kind of server that operate on behalf of multiple tenants (either simultaneously or sequentially?) and spawn pods?